internal/cypress/docker/runner.go

```diff
@@ -83,6 +85,11 @@ func (r *Runner) setup() error {
 		return err
 	}
 
+	// Warn the user regarding empty image name property
+	if r.Project.Docker.Image.Name == "" {
```

Comment:
@alexplischke what do you this about showing a warning message about missing docker image property?